apis/config/v1alpha1/configuration.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import "sigs.k8s.io/kueue/pkg/util/pointer"
+
+// SetConfigurationDefaults sets default values for ComponentConfig.
+func SetConfigurationDefaults(cfg *Configuration) {
```

Comment:
```suggestion
func SetDefaults_Configuration(cfg *Configuration) {
```

to match https://github.com/kubernetes/kubernetes/blob/09e8339ae4e575ec40e8ed0fc2829fcb678c0f19/pkg/scheduler/apis/config/v1beta3/defaults.go#L103

Unless the linter complains? Then feel free to remove the underscore.

Also, let's name the file `defaults.go`